### PR TITLE
Fixing the Python benchmark graph Layout

### PR DESCRIPTION
**Pull request by Cvr421**
The layout of the python benchmark graph is not properly align. It is overflowing to the other component. 
Fixing issue #4170

Issue 
image

Expected Behavior
Screenshot from 2024-03-18 15-01-34

### CODE REVIEW

**Comment by Cvr421** [comment]
@cartermp @chalin I have created the PR please review and let me know thoughts . Thanks

File: content/en/docs/languages/python/benchmarks.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ cSpell:ignore: Elems rrggbbaa
 
 <link rel="stylesheet" href="/css/benchmarks.css">
 
+<style>
+   @media (min-width: 100px) {
+     h3{
+         font-size: 0.9rem;
+         font-weight: bold;
+      }
+    }
```
Comment (svrnm) [inline content/en/docs/languages/python/benchmarks.md:15]: Can you make the css specific to the benchmark pages so it is not affecting any other page? 

If I understand Problem and solution correctly the titles are too long non-breaking words? Is there a way to force breaking them or change the word breaking behavior?

**Review by svrnm — CHANGES_REQUESTED** [review]
Thanks, I left a comment, please take a look.

File: content/en/docs/languages/python/benchmarks.md (hunk 1 above)
Comment (Cvr421) [reply content/en/docs/languages/python/benchmarks.md]: @svrnm i Have made the changes requested by you please review .

**Review by ocelotl — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Pull request merged**